app/Http/Controllers/VoucherController.php

```diff
@@ -186,7 +183,7 @@ public function createCash(CashVoucherRequestStore $request)
             TransactionFlowStatus::DONE->value
         );
         DB::commit();
-        $voucher->notify(new RequestCashVoucherForApprovalNotification(auth()->user()->token, $voucher));
+        $voucher->notify(new RequestVoucherForApprovalNotification(auth()->user()->token, $voucher));
```

Comment:
wrong way to notify na ni tom. auth()->user()->notify

Reply:
done